particle.inf

```diff
@@ -235,6 +235,11 @@ DEVICE_DESCRIPTION_P1_3 ="P1 Serial"
 DEVICE_DESCRIPTION_ELECTRON_1 = "Electron Serial"
 DEVICE_DESCRIPTION_ELECTRON_2 = "Electron USBSerial1"
 DEVICE_DESCRIPTION_ELECTRON_3 = "Electron Serial"
+DEVICE_DESCRIPTION_P1_1 ="Duo Serial"
```

Comment:
lol. I shouldn't use the Atom editor...I find the grey on black hard to read :-)